File: Module 2 - Loops (Iteration)/for loop/Challenge Problem/studentAttendence.cpp
/*Student Attendance
There are 10 students.
Print:
Student 1 Present

Student 2 Present

...

Student 10 Present
using a for loop.*/

#include<iostream>
using namespace std;

int main(){
    cout <<"\nStudent Attendence\n";
    for(int i=1;i<=10;i++){
        cout << "student "<<i<<" present\n";
    }

    cout <<"\nPrint only even-numbered students:\n";
    for(int i=2;i<=10;i+=2){
        cout << "student "<<i<<" present\n";
    }

    cout <<"\nPrint only odd-numbered students:\n";
    int j=0;
    for(int i=1;i<=10;i+=2){
        cout << "student "<<i<<" present\n";
    }
return 0;
}

File: Module 2 - Loops (Iteration)/for loop/Challenge Problem/studentAttendence.py
"""Student Attendance
There are 10 students.
Print:
Student 1 Present

Student 2 Present

...

Student 10 Present
using a for loop."""

print("Student Attendence")
for i in range(1,11):
    print("student ",i," present")

print("Print only even-numbered students:")
for i in range(2,11,2):
    print("student ",i," present")

print("Print only odd-numbered student")
for i in range(1,11,2):
    print("student ",i," present")